src/components/select/select.scss

```diff
@@ -228,3 +228,41 @@ md-optgroup {
     border: 1px solid #fff;
   }
 }
+
+md-select-menu[multiple] {
+    md-option {
+        height: $select-option-height;
+        padding: 0 $select-option-padding 0 ($select-option-padding / 2);
+        padding-left: ($select-option-padding * 2) + ($select-option-padding / 2);
+
+        &:before {
```

Comment:
I'm just a bit curious, what happens, when we change something in the `md-checkbox` style, then I guess, mostly the *mirror* checkbox will be forgotten to update.